.github/workflows/scripts/unix_test.sh

```diff
@@ -50,7 +59,6 @@ else
     if [[ $TI_WANTED_ARCHS == *"opengl"* ]]; then
         python3 tests/run_tests.py -vr2 -t4 -k "not torch" -a opengl
     fi
-    # Run metal and vulkan separately so that they don't use M1 chip simultaneously.
     if [[ $TI_WANTED_ARCHS == *"metal"* ]]; then
```

Comment:
This is no longer needed ;) We won't test metal backend on normal self-hosted gpu runners.

Reply:
right ;/